src/main/java/org/springframework/data/map/repository/config/MapRepositoryConfigurationExtension.java

```diff
@@ -83,31 +86,32 @@ protected AbstractBeanDefinition getDefaultKeyValueTemplateBeanDefinition(
 		return ParsingUtils.getSourceBeanDefinition(builder, configurationSource.getSource());
 	}
 
-	@SuppressWarnings({ "unchecked", "rawtypes", "NullAway" })
-	private static Class<? extends Map> getMapTypeToUse(RepositoryConfigurationSource source) {
+	private static Object getKeySpaceStore(RepositoryConfigurationSource source) {
+
+		Optional<String> keySpaceStoreRef = source.getAttribute("keySpaceStoreRef", String.class);
 
-		return (Class<? extends Map>) getAnnotationAttributes(source).get("mapType");
+		return keySpaceStoreRef.map(beanName -> new RuntimeBeanReference(beanName, KeySpaceStore.class)) //
```

Comment:
I meant to ask (in general) the ending the line w/ `//` I am guessing makes the formatter not reformat the lines smashed onto a single line and respects the multi-line fluent style that is soothing on the eyes. Is this correct?

Reply:
trailing comments force the formatter to keep line breaks. I know that's a style one has to get used to. The motivation is that we want our formatter to generally reformat linebreaks unless we explicitly want to take control. NOFORMAT blocks apply to all aspects of formatting and that is too much exclusion for aspects that  matter.